iina/MainMenuActions.swift

```diff
@@ -90,11 +90,10 @@ extension MainMenuActionHandler {
   }
 
   @objc func menuStep(_ sender: NSMenuItem) {
-    let seconds = Double(abs((sender.representedObject as? Int) ?? 5))
-    if sender.tag == 0 { // -> 5s
-      player.seek(relativeSecond: seconds, option: .relative)
-    } else if sender.tag == 1 { // <- 5s
-      player.seek(relativeSecond: -seconds, option: .relative)
+    if let args = sender.representedObject as? (Double, Preference.SeekOption) {
+      player.seek(relativeSecond: args.0, option: args.1)
+    } else {
+      player.seek(relativeSecond: 5, option: Preference.SeekOption.defaultValue)
```

Comment:
You forgot to consider `sender.tag`, if `sender.tag == 1` the `relativeSecound` should be -5. When the menu item is not matched by keybindings, we will have forward and **backward** menu items by default.